latch_cli/services/register/register.py

```diff
@@ -185,6 +189,7 @@ def register(
     pkg_root: str,
     disable_auto_version: bool = False,
     remote: bool = False,
+    yes: bool = False,
```

Comment:
rename this to `disable_confirmation` or something else more descriptive than `yes`